src/main/java/edu/kis/vh/nursery/dataStructures/IntArrayStack.java

```diff
@@ -1,10 +1,10 @@
 package edu.kis.vh.nursery.dataStructures;
 
-public class IntArraySTack implements IntArraySTackInterface {
+public class IntArrayStack implements Structrues {
 
     private static final int ARRAY_CAPACITY = 12;
 
-    private static final int FULL_RHYMER_INDICATOR = 11;
+    private static final int FULL_RHYMER_INDICATOR = 12;
```

Comment:
Still: This constant is related to ARRAY_CAPACITY. Changing only one leads to an error (inconsistency). You should use one in the definition of the other (not change the value).